blocks/contact-form/contact-form.js

```diff
@@ -1,13 +1,18 @@
+import submitContactInfo from './submit-contact-info.js';
+
 // Selectors
 const blockChildDivs = document.querySelectorAll('.contact-form > div');
 
 export default function decorate(block) {
```

Comment:
a valid HTML form should be wrapped in a `form` element and I don't see that tag anywhere. is that also part of the commented code?

it's not just validity/syntax sugar/tag clutter either. if you wrap all inputs in a form, there's an additional benefit:
you can call `.elements` on the form and get access to all the input elements within the form in a programical way. see https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements

that way, you can map the form's inputs and values to a JSON in a few lines of code in a more generic way.